buildpack/telemetry/fluentbit.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import os
+import subprocess
+import shutil
+import socket
+
+import backoff
+
+from buildpack import util
+
+
+NAMESPACE = "fluentbit"
+CONF_FILENAME = f"{NAMESPACE}.conf"
+FILTER_FILENAME = "redaction.lua"
+FLUENTBIT_ENV_VARS = {
+    "SPLUNK_PORT": os.getenv("SPLUNK_PORT", default="8088"),
+    "FLUENTBIT_LOGS_PORT": os.getenv("FLUENTBIT_LOGS_PORT", default="5170"),
+}
+
+REQUIRED_FLUENTBIT_ENV_VARS = ["SPLUNK_HOST", "SPLUNK_TOKEN"]
```

Comment:
@denis-shuvalov @pappachino splunk and fluentbit are not bound to each other. We'll have use cases where we use fluentbit for other integrations in the future, for which this code will need to be massively restructured. Can we handle this right away to make life easier in the future?

Thoughts?

Reply:
Split the fluentbit file for splunk and fluentbit dedicated ones.